docker-compose.yml

```diff
@@ -38,6 +38,8 @@ services:
       "--appendonly",
       "yes"
     ]
+    ports:
```

Comment:
I actually find this really useful for dev/debugging -- i spin up rekor in docker compose, run a `docker stop rekor_server` and then i can run the server on the command line via

```
go run ./cmd/rekor-server serve
```

It still interacts with the redis/trillian containers bc of the port forwarding and it's faster than having to rebuild the rekor server container every time I want to test a change.

I actually left it in here on purpose in case others do the same!